projects/swimlane/ngx-ui/src/lib/components/dialog/large-format/components/large-format-dialog-header-title/large-format-dialog-header-title.component.scss

```diff
@@ -0,0 +1,26 @@
+@import 'colors/variables';
+
+.ngx-large-format-dialog-header-title {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+  flex: 0 0 20%;
+  max-width: 50%;
+
+  & h1,
```

Comment:
Nitpick: do we need this &? Removing would be the same selector right?

Reply:
It would, but I tend to keep `&` for consistency purposes. 

```css
&.some-class {}
& .some-class {}

// vs

.some-class {}
&.some-class {}
```

Reply:
1st one will work. Just keeping consistent was all. 👍